waspc/README.md

```diff
@@ -39,16 +39,16 @@ the project. The best way to install it is via
 is a convenient way of installing and selecting versions of `cabal`, `hls` and
 `ghc`.
 
+:warning: If you are on a Mac, we still recommend you use the official [ghcup](https://www.haskell.org/ghcup/) installer over Homebrew, as the initial setup will be easier, and you won't have to worry about setting the correct environment variables.
```

Comment:
I've added this bit, because I tried to install ghcup through Homebrew, and the env vars were all wrong.